components/automate-ui/src/app/pages/project/rules/project-rules.component.ts

```diff
@@ -111,13 +114,20 @@ export class ProjectRulesComponent implements OnInit, OnDestroy {
         id: rule_id,
         project_id
       }));
-    });
+   });
 
     this.store.select(projectFromRoute).pipe(
       filter(identity),
       takeUntil(this.isDestroyed)
     ).subscribe(project => {
       this.project = project;
+      this.authorizedChecker.setPermissions([
```

Comment:
This does in code what `<app-authorized>` does in the template.

Reply:
thats a nifty trick 💯